app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    _, src, dest = command.split()
```

Comment:
The `command.split()` assumes that the command string is always correctly formatted with exactly three parts. Consider adding error handling to manage cases where the command does not meet this expectation.